ragmac_xdem/dem_postprocessing.py

```diff
@@ -181,6 +192,7 @@ def _stats_wrapper(dem_path):
     )
 
     # Save output to file
+    df_stats.dropna(inplace=True)
```

Comment:
I this what we want? Would it not be better to keep NaNs where appropriate, so we know that particular DEM is useless (rather than not processed)?

Reply:
I agree, but I think we  only use the valid paths in the from coreg onwards, with the modification in utils.py here:
![image](https://user-images.githubusercontent.com/29011666/158296319-cd19860a-6534-47b2-9922-a428be52c81c.png)

I will be happy to change, if that is still preferred :)

Reply:
I was thinking, in the saved CSV file, it's good to have the DEM listed, even if it has no valid data. Anyway, it's not big deal.